codegen-winter/src/air/mod.rs

```diff
@@ -28,7 +35,10 @@ fn add_air_struct(scope: &mut Scope, name: &str) {
     scope
         .new_struct(name)
         .vis("pub")
-        .field("context", "AirContext<Felt>");
+        .field("context", "AirContext<Felt>")
+        .field("stack_inputs", "Vec<Felt>")
+        .field("stack_outputs", "Vec<u64>")
+        .field("overflow_addrs", "Vec<u64>");
```

Comment:
the base implementation of the public inputs struct needs to depend on the public inputs that are defined in the original Air file. They shouldn't be hard-coded

Reply:
Fixed.